components/status_dropdown/status_dropdown.tsx

```diff
@@ -423,12 +431,16 @@ export default class StatusDropdown extends React.PureComponent<Props, State> {
                     listId={'status-drop-down-menu-list'}
                 >
                     {currentUser && (
-                        <Menu.Header>
+                        <Menu.Header onClick={this.openProfileModal}>
                             {this.renderProfilePicture('lg')}
                             <div className={'username-wrapper'}>
-                                <Text margin={'none'}>{`${currentUser.first_name} ${currentUser.last_name}`}</Text>
+                                <Text
+                                    className={'bold'}
+                                    margin={'none'}
+                                >{`${currentUser.first_name} ${currentUser.last_name}`}</Text>
```

Comment:
Should we reformat this?
```suggestion
                                <Text
                                    className={'bold'}
                                    margin={'none'}
                                >
                                    {`${currentUser.first_name} ${currentUser.last_name}`}
                                </Text>
```